### PR TITLE
docs(drash/v2.x): Add docs for etag service

### PR DESCRIPTION
**Pull request by ebebbington**
Adds docs for https://github.com/drashland/drash/pull/568

### CODE REVIEW

File: docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/6_etag.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+});
+
+// Create your resource
+
+class HomeResource extends Drash.Resource {
+
+  public paths = ["/"];
+
+  // Tell the resource what HTTP methods should have an Etag header set. In this
+  // case, we are telling the resource to set it on the POST method. This means
```
Comment (crookse) [inline docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/6_etag.md:65]:
```suggestion
  // case, we are telling the resource to set it on the GET method. This means
```

**Review by crookse — CHANGES_REQUESTED** [review]

**Review by crookse — APPROVED** [review]
i made some revisions. otherwise, LGTM

File: docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/etag.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# ETag
```
Comment (ebebbington) [inline docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/etag.md:1]: Only problem with this is the service is still named `EtagService`, not `ETagService`, do we rename it in drash or rename it here?

Comment (crookse) [reply docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/etag.md:1]: https://github.com/drashland/drash/commit/c2f6831ef9640919daca78b4e63a3a3b5e1373cc

Comment (crookse) [reply docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/etag.md:1]: i changed it 😬

**Pull request merged**